code/stack.yaml

```diff
@@ -94,6 +94,8 @@ extra-deps:
 - unix-2.8.5.1@sha256:3f702a252a313a7bcb56e3908a14e7f9f1b40e41b7bdc8ae8a9605a1a8686f06,9808
 - Cabal-3.12.1.0@sha256:08be296ddd941b3f9be4bd125fbb3e6857f5c707b8d10464f6f1d30c91ca3e5f,13551
 - Cabal-syntax-3.12.1.0@sha256:6dbb06fb08ff77520947fb4c1ef281c9cea5b8dc7fd9a41ad62273fa916cf4b2,7407
+- time-1.12.2@sha256:88e8493d9130038d3b9968a2530a0900141cd3d938483c83dde56e12b875ebc8,6510
+- Win32-2.14.0.0
```

Comment:
@daijingz is on a 32bit machine. I wouldn't be surprised if that is the discrepancy between your two machines.

Considering [Win32](https://hackage.haskell.org/package/Win32) is an official library of Haskell.org, I think it's okay to add it without too much concern. That being said, we should also pin the checksum in this line.

Reply:
I can confirm that I had this same issue, even though I'm on a 64-bit machine (I can drop more specs if that would help).

The specific line that was recommended to me was the following (my time-1.12.2 line is the same as @daijingz's):
```yaml
- Win32-2.14.0.0@sha256:e34af84fec733b5c0c8f052ec39499785e719e2fbbe308983adf26c82ea3704d,5942
```

Reply:
Thanks for making this more precise. So I'll accept this one it is pinned.

Reply:
Updated.